gui/src/app/pyodide/AnalysisPyFileEditor.tsx

```diff
@@ -168,10 +183,27 @@ const AnalysisPyFileEditor: FunctionComponent<Props> = ({
       onSetEditedText={setEditedFileContent}
       readOnly={readOnly}
       toolbarItems={toolbarItems}
+      contentOnEmpty={contentOnEmpty}
     />
   );
 };
 
+const analysisPyTemplate = `import matplotlib.pyplot as plt
+
+# Get the parameter names
+pnames = draws.parameter_names
+
+# Show histogram for first parameter
+for pname in pnames:
+    print(pname)
+    samples = draws.get(pname)
+    print(samples.shape)
+    plt.hist(samples.ravel(), bins=30)
+    plt.title(pname)
+    plt.show()
+    break
```

Comment:
I understand the intention, but I think it is confusing to write in a loop that breaks. The first parameter name will always be `lp__`, so maybe:

```python
# Get the parameter names
print(draws.parameter_names)

# plot the lp parameter
samples = draws.get("lp__")
print(samples.shape)
plt.hist(samples.ravel(), bins=30)
plt.title("lp__")
plt.show()

Reply:
Done